### ISSUE TITLE
SwiftBar output obscuring application menubar options

### ISSUE THREAD

**Issue by nickjvturner** [swiftbar/SwiftBar#155]
SwiftBar Version 1.2.2 (252) appears to have introduced an issue where SwiftBar output will appear on top of active application menubar options.
<img width="1440" alt="Screenshot 2021-02-01 at 21 38 31" src="https://user-images.githubusercontent.com/34604855/106521567-519db100-64d6-11eb-802e-7248f5e8f7fa.png">

**Comment by melonamin** [comment]
Can you reproduce it reliably? I can't (first four items are from Swiftbar)

https://user-images.githubusercontent.com/222100/106525042-60be3800-64b1-11eb-9224-8c0fb141c0a8.mov

Overall looks like an OS glitch, especially if you can't reproduce it, items in menu bar are placed and hidden by OS, Swiftbar has no control over it

**Comment by nickjvturner** [comment]
Let me try again tomorrow.

It's the first time I've seen this behaviour. Will report back if I can reproduce.

**Comment by nickjvturner** [comment]
No, I cannot reproduce.
